wagtailmarkdown/mdx/tables/__init__.py

```diff
@@ -14,9 +14,19 @@
 Copyright 2009 - [Waylan Limberg](http://achinghead.com)
 """
 
+
+import markdown
+from packaging.version import Version

```

Comment:
Wasn't sure how this would play with non-pip solutions (e.g. Conda) but it looks like Conda projects can use pip when a package is not available in the Conda repos.

From my research it seem it should all be fine. Happy to go with this.